### DISCUSSION TITLE
[PasswordInput] add aria-label in rightSection

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#1927] [Feature requests]
## Suggest

I'm stepping through the components in order to understand Mantine. In the meantime, I have a suggestion about `<password-input>`.

And if this proposal is accepted, I will create a PR to address it!

## Description

![image](https://user-images.githubusercontent.com/27342882/181863658-63038035-4907-4e69-a875-42feb8561ae3.png)

## What

I think should add the `aria-label` attribute to the highlighted area in the photo and `toggle password visibility` as the value.

```tsx
// PasswordInput.tsx
const rightSection = (
    <ActionIcon<'button'>
      className={classes.visibilityToggle}
      tabIndex={toggleTabIndex}
      radius={radius}
      size={theme.fn.size({ size, sizes: buttonSizes })}
      aria-label='toggle password visibility' // here
      unstyled={unstyled}
      onMouseDown={(event) => {
        event.preventDefault();
        toggle();
      }}
      onKeyDown={(event) => {
        if (event.key === ' ') {
          event.preventDefault();
          toggle();
        }
      }}
    >
      <VisibilityToggleIcon reveal={reveal} size={theme.fn.size({ size, sizes: iconSizes })} aria-hidden /> // here
    </ActionIcon>
  );
```

## Why

This is because it is thought that it will be possible to convey information about the element to the browser and to deliver meaningful information to users using the web through screen readers.

**Comment by rtivital** [comment]
Thanks for feature request, I'll add aria-label for visibility toggle button in next patch.

**Reply by rtivital** [reply]
Added in 5.0.2
